packages/syft/src/syft/gevent_patch.py

```diff
@@ -37,6 +34,6 @@ def is_notebook() -> bool:
 
 jupyter_notebook = is_notebook()
 
```

Comment:
so I had to remove this to get rid of threading problems. There is a much longer story here, but I suspect we have to dedicate maybe 1 or 2 weeks to come up with a proper multithreading/processing plan